insitu/views/user_records.py

```diff
@@ -7,11 +8,27 @@
     ProductRequirement,
     Requirement,
     User,
+    LoggedAction
 )
 from insitu.views import ProtectedTemplateView
 from insitu.views.protected import (
     IsCurrentUser,
 )
+from datetime import date, datetime
+
+
+def export_logs(request):
```

Comment:
Please use class based views. Maybe add a form class to manipulate the form data.